testcases/TCID-VMWx-Portal-GUI-AccountSettings.md

```diff
@@ -0,0 +1,126 @@
+# GUI Test Cases For My Account Settings in Litmus-Portal
```

Comment:
We are now not using `VMWx` can we replace all VMVx with EKS in all places that are gitlab-ci.yaml, master-plan.yaml, and here for me files